keyboards/thedogkeyboard/readme.md

```diff
@@ -0,0 +1,15 @@
+# thedogkeyboard
+
+![thedogkeyboard](https://i.gyazo.com/70ac677c1e75c04b812e5dce311f7901.jpg)
+
+1ProMicro 100% Keyboard.
+
+Keyboard Maintainer: [kakunpc](https://github.com/kakunpc)  
```

Comment:
We recently changed the template for this to a list. No more concerns about trailing spaces.

```suggestion
* Keyboard Maintainer: [kakunpc](https://github.com/kakunpc)
```